CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Not sure we want to go down a version like this, @merlinvn any thoughts?

Reply:
Basicly, no need to go up to Version 3.10